blueprints/addon/index.js

```diff
@@ -1,21 +1,23 @@
-var fs = require('fs-extra');
-var existsSync = require('exists-sync');
-var path = require('path');
-var walkSync = require('walk-sync');
-var stringUtil = require('ember-cli-string-utils');
-var uniq = require('ember-cli-lodash-subset').uniq;
-var SilentError = require('silent-error');
-var sortPackageJson = require('sort-package-json');
-var date = new Date();
-
-var normalizeEntityName = require('ember-cli-normalize-entity-name');
-var stringifyAndNormalize = require('../../lib/utilities/stringify-and-normalize');
+'use strict';
+
+const fs = require('fs-extra');
+const existsSync = require('exists-sync');
+const path = require('path');
+const walkSync = require('walk-sync');
+const stringUtil = require('ember-cli-string-utils');
+const uniq = require('ember-cli-lodash-subset').uniq;
+const SilentError = require('silent-error');
+const sortPackageJson = require('sort-package-json');
+let date = new Date();
```

Comment:
@stefanpenner - The linting rules absolutely still allow us to use `const` there, but I used the basic heuristic of "if its a `require` then make it `const`" to automate these changes and this one just ended up being a `let`.

Once we land this we should start reviewing "whats left" and fixing things like this one...